values.yaml

```diff
@@ -308,6 +308,7 @@ loadBalancer:
 db: 
   name: hyperswitch
   host: db_host
+  replica_host: replica_db_host
```

Comment:
what if the replica has different username and password? is that scenario not valid?